scooter/src/app.rs

```diff
@@ -526,7 +616,19 @@ impl App {
             None => current_dir().unwrap(),
         };
 
-        let search_fields = SearchFields::with_default_values().with_advanced_regex(advanced_regex);
+        let mut disable_populated_fields = false;
+        if config
+            .editor_open
+            .as_ref()
+            .map(|c| c.disable_populated_fields)
+            .unwrap_or(false)
+        {
+            disable_populated_fields = true
+        }
+
```

Comment:
I'm not sure about this code.

The AI overlords  suggested I could make it like this... but I think "if" clearly expresses what is happening here and doesn't hide anything.

``` rust
let mut disable_populated_fields = config.editor_open.map(|c| c.disable_populated_fields).unwrap_or(false);
```